anitya/tests/db/test_models.py

```diff
@@ -516,6 +549,7 @@ def test_init_distro(self):
         self.assertEqual(distros[0].name, 'Debian')
         self.assertEqual(distros[1].name, 'Fedora')
 
+    # TODO: Split to separate tests
```

Comment:
I recommend filing a ticket in your issue tracker over leaving a # TODO comment.

Reply:
I created a new issue for this https://github.com/release-monitoring/anitya/issues/680 and I will remove the TODO comment.